### PR TITLE
Draft of blog post announcing 1.1

### PR DESCRIPTION
**Pull request by oaktowner**
Blog post announcing 1.1. Still needs links to docs, release notes, and actual download.

### CODE REVIEW

File: content/blog/2019/announcing-1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+We are grateful to everyone who has worked hard on Istio over the last few
+months -- patching 1.0, adding features 1.1, and, lately, doing tons of
+testing on 1.1. Thanks especially to those companies who worked with us
+installing and upgrading to the early builds and helping us catch problems
+before the release.
+
+So: now’s the time! Grab 1.1, check out the updated documentation, install
+it and...happy meshing!
+
+We're pleased to announce the availability of Istio 1.0.6. Please check out the [release notes](/about/notes/1.0.6/) to learn more and download this update.
```
Comment (rshriram) [inline content/blog/2019/announcing-1.1/index.md]: please remove last line..

File: content/blog/2019/announcing-1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+(Thursdays at 11am PT) and in the
+[Working
+Groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+And if you haven’t yet joined the conversation at
+[discuss.istio.io](https://discuss.istio.io), head over, log in with your
+Github credentials and join us!
+
+We are grateful to everyone who has worked hard on Istio over the last few
+months -- patching 1.0, adding features 1.1, and, lately, doing tons of
+testing on 1.1. Thanks especially to those companies who worked with us
```
Comment (rshriram) [inline content/blog/2019/announcing-1.1/index.md]: companies and end users

File: content/blog/2019/announcing-1.1/index.md (hunk 1 above)
Comment (geeknoid) [inline content/blog/2019/announcing-1.1/index.md]: Leftovers...

File: content/blog/2019/announcing-1.1/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Istio 1.1
+description: Istio 1.1 release announcement.
+publishdate: 2019-03-14
+attribution: The Istio Team
+weight: 77
```
Comment (geeknoid) [inline content/blog/2019/announcing-1.1/index.md]: Please remove weight: here, we don't need this anymore.

File: content/blog/2019/announcing-1.1/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+We have also improved the multicluster capabilities and usability. We
+listened to the community and improved defaults for traffic control and policy.
+We introduced a new component called Galley. Galley validates that sweet, sweet
+YAML, reducing the chance of configuration errors. Galley will also be
+instrumental in multicluster setups, gathering service discovery information
+from each Kubernetes cluster. We are also supporting additional multicluster
+topologies including single control plane and multiple synchronized control
+planes without requiring a flat network.
+
+There is lots more -- see release notes here.
```
Comment (geeknoid) [inline content/blog/2019/announcing-1.1/index.md]: [release notes](/docs/about/notes/1.1/)

File: content/blog/2019/announcing-1.1/index.md (hunk 1 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: ack!

File: content/blog/2019/announcing-1.1/index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+There is more going on in the project as well. We know that Istio has a lot of
+moving parts and can be a lot to take on. To help address that, we recently
+formed a Usability Working group (feel free to join). There is also a lot
```
Comment (geeknoid) [inline content/blog/2019/announcing-1.1/index.md]: [Usability Working Group](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)

File: content/blog/2019/announcing-1.1/index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Istio 1.1
+description: Istio 1.1 release announcement.
+publishdate: 2019-03-14
+attribution: The Istio Team
+weight: 77
+---
+We are pleased to announce the release of Istio 1.1.
+
```
Comment (geeknoid) [inline content/blog/2019/announcing-1.1/index.md]: Let's insert some buttons here pointing directly to the 1.1 release notes and the 1.1 download page.

I'll add the needed support in the site infra to enable that.

File: content/blog/2019/announcing-1.1/index.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+Meeting](https://github.com/istio/community#community-meeting)
+(Thursdays at `11 a.m.`) and in the
+[Working
+Groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+And if you haven’t yet joined the conversation at
+[discuss.istio.io](https://discuss.istio.io), head over, log in with your
+GitHub credentials and join us!
+
+We are grateful to everyone who has worked hard on Istio over the last few
+months -- patching 1.0, adding features 1.1, and, lately, doing tons of
```
Comment (frankbu) [inline content/blog/2019/announcing-1.1/index.md]: adding features to 1.1

File: content/blog/2019/announcing-1.1/index.md (hunk 1 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: ack fixed

File: content/blog/2019/announcing-1.1/index.md (hunk 3 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: ack fixed

File: content/blog/2019/announcing-1.1/index.md (hunk 4 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: I linked it to /notes/about/1.1/ which seems appropriate.

File: content/blog/2019/announcing-1.1/index.md (hunk 5 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: ack

File: content/blog/2019/announcing-1.1/index.md (hunk 7 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: ack

File: content/blog/2019/announcing-1.1/index.md (hunk 6 above)
Comment (oaktowner) [reply content/blog/2019/announcing-1.1/index.md]: I think this is the only thing holding up the post.  I've got the CTA later on...are you sure we need these buttons?

**Comment by geeknoid** [comment]
It's fine to push as-is. The buttons are nice though, I'll add them afterwards.

/lgtm
/approve

**Pull request merged**